docs/machine-learning/tutorials/sentiment-analysis.md

```diff
@@ -68,9 +68,9 @@ Predict the **sentiment** of a new website comment, either positive or negative.
 
     In Solution Explorer, click on the **Solutions and Folders** icon. Right-click on the *bin* folder, select **Add** > **New Folder**. Type "Data" and hit Enter. Click again on the **Solutions and Folders** icon to return to the solution view.
 
-3. Install the **Microsoft ML.NET NuGet Package**:
+3. Install the **Microsoft.ML NuGet Package**:
 
-    In Solution Explorer, right-click on your project and select **Manage NuGet Packages**. Choose "nuget.org" as the Package source, select the Browse tab, search for **Microsoft ML.NET**, select that package in the list, and select the **Install** button. If prompted to select a package management format, select **PackageReference in project file**.
+    In Solution Explorer, right-click on your project and select **Manage NuGet Packages**. Choose "nuget.org" as the Package source, select the Browse tab, search for **Microsoft.ML**, select that package in the list, and select the **Install** button. Select the **OK** button on the **Preview Changes** dialog and then select the **I Accept** button on the **License Acceptance** dialog if you agree with the license terms for the packages listed.
```

Comment:
@mairaw it looks that using **PackageReference** is essential: https://github.com/dotnet/machinelearning/issues/93

Reply:
Thanks for confirming @eerhardt. So it's fine to remove that step since the tutorial is using a .NET Core app.